add-domain

```diff
@@ -1093,3 +1093,5 @@ meldung-vols.online
 beritapb.com
 sgp1.digitaloceanspaces.com
 cpanel01wh.bkk1.cloud.z.com
+naps.top
```

Comment:
Chose one of them....

If you will blacklist by wildcard (domain + sub-doamins), please use https://github.com/mitchellkrogza/phishing/blob/main/add-wildcard-domain